README.md

```diff
@@ -18,6 +18,10 @@ Use the community resources below for getting help with the ADOT Collector.
 * If you think you may have found a bug, open a [bug report](https://github.com/aws-observability/aws-otel-collector/issues/new?template=bug_report.md).
 * For contributing guidelines, refer to [CONTRIBUTING.md](CONTRIBUTING.md).
 
+### Notice: ADOT Collector v0.32.0 Breaking Change
```

Comment:
There is no breaking change in v0.32.0. The breaking change could be introduced in v0.33.0 at the earliest.